src/main/java/Duke.java

```diff
@@ -1,10 +1,103 @@
+import java.io.InputStreamReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+// import java.util.Scanner;
```

Comment:
Can remove the comment here if you don't intend to use this.

Reply:
Removed. Thank you!